app/services/journal_row_converter.rb

```diff
@@ -0,0 +1,24 @@
+class JournalRowConverter
+
+  # Returns an array of generated journal_rows given a single order_detail.
+  def self.from_order_detail(order_detail, journal = nil)
+    [{
+      account: order_detail.product.account,
```

Comment:
This didn't make sense to me either.  As well as how we set account within `from_product` (I think we simply use a string?).